CHANGELOG-5.1.md

```diff
@@ -0,0 +1,12 @@
+CHANGELOG for 5.1.x
```

Comment:
this file should be removed: the main changelog is autogenerated
instead, please update the changelog of the component

Reply:
Done.